wms_connector/models/synchronize_importable_mixin.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import datetime
+
+from odoo import fields, models
+
+
+class SynchronizeImportableMixin(models.AbstractModel):
+    _name = "synchronize.importable.mixin"
```

Comment:
I don't know where this mixin is make it concrete. It should be pickings ?

Reply:
for public discourse: 

can be applied to pickings, inventories and whatever else